lib/structurely/entities/base_entity.rb

```diff
@@ -0,0 +1,50 @@
+module Structurely
+  module Entities
+    EntityResponse = Struct.new(:metadata, :results)
+
+    # Workaround to silence Hashie warnings logger
+    class BaseEntityHashie < Hashie::Mash
+      disable_warnings
+    end
+
+    class BaseEntity < ::ApiStruct::Entity
```

Comment:
Docs to describe what this does, such as what input is expected and what output is generated?